LICENSE

```diff
@@ -13,7 +13,7 @@ Business Source License 1.1
 
 Licensor:                  Materialize, Inc.
 
-Licensed Work:             Materialize Version 20240925
+Licensed Work:             Materialize Version v0.118.1-dev.0
```

Comment:
Why is this not using the date now? Maybe we should also change it in the daily license-update job

Reply:
No idea, that's the `bin/bump-version` script's doing!

Reply:
Reverted.